helix-term/src/keymap/default.rs

```diff
@@ -26,6 +26,14 @@ pub fn default() -> HashMap<Mode, Keymap> {
         "home" => goto_line_start,
         "end" => goto_line_end,
 
+        "A-w" => { "Alter Window"
+            "A-w"|"A-left" => shrink_buffer_width,
+            "A-l"|"A-right" => grow_buffer_width,
+            "A-j"|"A-down" => shrink_buffer_height,
+            "A-k"|"A-up" => grow_buffer_height,
+            "A-f" => buffer_expand_mode,
+        },
```

Comment:
I think it would be handy to have also a sticky mode for this (see `Z`) so you don't have to hold alt the whole time

Reply:
Thanks for the tip, didn't realise that was a thing. It's now much nicer to use!